### PR TITLE
skip auth setup when developing e2e tests locally

### PR DESCRIPTION
**Pull request by fungairino**
## What does this PR do?

- Makes local development faster by skipping auth if the auth file is still available locally

## Checklist

- [ ] Add jest or playwright tests and/or storybook stories
- [X] Designate a primary reviewer @fregante

### CODE REVIEW

**Review by fregante — APPROVED** [review]
A couple of nits code-wise

File: end-to-end-tests/fixtures/extensionBase.ts [hunk 1]
```diff
@@ -43,10 +43,7 @@ process.env.PW_CHROMIUM_ATTACH_TO_OTHER = "1";
 const getStoredCookies = async (): Promise<Cookie[]> => {
   let fileBuffer;
   try {
-    fileBuffer = await fs.readFile(
-      // eslint-disable-next-line unicorn/prefer-module -- TODO: import.meta.dirname throws "cannot use 'import meta' outside a module"
-      path.join(__dirname, "../.auth/user.json"),
-    );
+    fileBuffer = await fs.readFile(path.join(__dirname, "../.auth/user.json"));
```
[the remark this answers is not in the document]
Comment (fungairino) [reply end-to-end-tests/fixtures/extensionBase.ts:46]: I'll give this another pass in the PR I'm about to submit

[the remark this answers is not in the document]
Comment (fregante) [reply end-to-end-tests/fixtures/extensionBase.ts:46]: Oh right. The import version should work then, that one is always relative to the current file.

File: .eslintrc.js [hunk 2]
```diff
@@ -194,6 +194,7 @@ module.exports = {
       rules: {
         "no-restricted-imports": "off",
         "unicorn/prefer-dom-node-dataset": "off",
+        "unicorn/prefer-module": "off", // `import.meta.dirname` throws "cannot use 'import meta' outside a module"
```
[the remark this answers is not in the document]
Comment (fungairino) [reply .eslintrc.js:197]: I think that section includes rules that only apply to jest tests. Ex: Getting this error:
```
ESLint: Avoid destructuring queries from `render` result, use `screen.getByRole` instead (testing-library/prefer-screen-queries)
```
on this:
`await page.getByRole("button", { name: "Log in" }).click();`

**Pull request merged**